CMakeLists.txt

```diff
@@ -557,6 +558,20 @@ if (DRM_INCLUDE_DIRS)
   endif()
 endif()
 
+# Habana headers
```

Comment:
All this habana support is not related to rdma-core tests which are intended to test code and devices supported by rdma-core.

Reply:
Hey, @rleon.
Pyverbs already have some HW accelerator support like Cuda or DRM for HW accelerators that support it (AFAIK from Linux kernel non relevant currently).
What about testing other HW accelerators that some providers might want to test using pyverbs ? perftest does support adding more HW accelerators to their testing interface.
This can be limiting because there are HW accelerators that we have support for except for CUDA like Amazon's Trainium or Habana's Gaudi.

Thanks.

Reply:
It's important to note that we don't test the accelerator itself but rdma-core - kernel - device integration with different accelerators.

Reply:
Existing tests don't test accelerators, but standard dmabuf interface.

Reply:
@rleon, yes exactly. Maybe I miss explained. I didn't add new tests for Habana only make the existing one work with other HW accelerator types.

Reply:
How do you test dmabuf without an accelerator?

Reply:
Hey @gal-pressman, we currently doesn't support dma buffers without HW accelerator and this patch is for supporting dma buffers on non-Cuda accelerators.

Reply:
@rleon How can we move forward with this? We would really like to enable dmabuf flow testing on platforms that don't have Nvidia accelerations.

Reply:
@YonatanNachum Do you mean DMA ring-buffer to fetch, or DMA data buffer to transmit between GPU?

Reply:
This PR is for testing of MR registration using `ibv_reg_dmabuf_mr` with various dmabuf exporters.